grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java

```diff
@@ -96,6 +96,15 @@
 
     VALIDATION_RULE_ILLEGAL_ATOMIC_IN_HEAD("The rule [%s] of type [%s] does not form a valid Horn clause, as its head contains illegal atomics\n"),
 
+    VALIDATION_RULE_INVALID_RELATION_TYPE("Attempting to define a rule containing a relation pattern with type [%s] which is not a relation type\n"),
+
+    VALIDATION_RULE_INVALID_RESOURCE_TYPE("Attempting to define a rule containing a resource pattern with type [%s] which is not a resource type\n"),
+
+    VALIDATION_RULE_RESOURCE_OWNER_CANNOT_HAVE_RESOURCE("Attempting to define a rule containing a resource pattern of type [%s] with owner type [%s] that cannot have this resource\n"),
```

Comment:
maybe just change `owner type` to `type`. `owner type` makes me think there is another type out there.